index.bs

```diff
@@ -21,11 +21,9 @@ Markup Shorthands: css no
 Logo: https://webmachinelearning.github.io/webmachinelearning-logo.png
 Deadline: 2023-10-01
 Status Text: <p>
-  Further implementation experience and user feedback is being gathered for the
-  {{MLCommandEncoder}} interface that proposes to enable more efficient WebGPU
-  integration. A proposal to
-  <a href="https://github.com/webmachinelearning/webnn/pull/322">simplify
-  MLContext creation</a> is being discussed. This document is maintained and
+  Since the <a href="https://www.w3.org/TR/2023/CR-webnn-20230330/">initial Candidate Recommendation Snapshot</a> the Working Group has gathered further <a href="https://webmachinelearning.github.io/webnn-status/">implementation experience</a> and added new operations and data types needed for well-known <a href="https://github.com/webmachinelearning/webnn/issues/375">transformers to support generative AI use cases</a>. In addition, the group has removed select features informed by this implementation experience: higher-level operations that can be expressed in terms of lower-level primitives in a performant manner, and support for synchronous execution. The group has also updated the specification to use modern authoring conventions to improve interoperability and precision of normative definitions and is developing a new feature, a <a href="https://github.com/webmachinelearning/webnn/issues/482">backend-agnostic storage type</a>, to improve performance and interoperability between the WebNN, WebGPU APIs and purpose-built hardware for ML.
```

Comment:
Looks good to me. Should we expand a bit more on what 'further implementation experience' is? i.e. should we be more specific about the ongoing work in the Chromium code base across Google, Microsoft, and Intel, etc.?

Reply:
Thanks @wchao1115 for your review. It may be hard to succinctly expand 'further implementation experience' in the limited space we have in this status section.

For this reason, we direct interested readers to https://webmachinelearning.github.io/webnn-status/ that also provides further pointers to Web Platform Tests. WPT is a resource W3C stakeholders are interested in in particular when they evaluate implementation experience. I think we are good in this respect with the materials we have.